.docs/user-guide/application.md

```diff
@@ -69,6 +69,10 @@ including but not limited the following examples.
         $ docker run -d --volume-driver=rexray -v mongodb:/data/db mongo
 
  * [MySQL](https://hub.docker.com/_/mysql/)
+ 
+         $ docker volume create --driver=rexray --name=mysqldemovolume --opt=size=<sizeInGB>
```

Comment:
@cantbewong,

Lines 73-74 have an incongruent amount of leading whitespace characters. Based on the other lines it appears line 73 should be have one less leading space.